exercise1.R

```diff
@@ -1,5 +1,27 @@
+library(ggplot2)
+
 ba = read.table("https://raw.githubusercontent.com/OxfordIHTM/teaching_datasets/main/ba.dat",header=TRUE)
 
+average = rowMeans(ba)
+ba$average = average
+#Mean differences 
+difference <- ba$Wright - ba$Mini
+ba$difference = difference
+mean_difference = mean(difference)
+
+#Confidence interval
+lower <- mean_difference - 1.96*sd(ba$difference)
+upper <- mean_difference + 1.96*sd(ba$difference)
+
+#Create Bland Altman Plot
+ggplot(ba, aes(x = average, y = difference)) +
+  geom_point() +
+  geom_hline(yintercept = mean_difference) +
+  geom_hline(yintercept = lower, color = "red", linetype="dashed") +
+  geom_hline(yintercept = upper, color = "red", linetype="dashed") +
+  ylab("PEFR (l/min) Mini Wright meter - Wright meter ") +
+  xlab("PEFR (l/min) Mini Wright meter + Wright meter / 2")
```

Comment:
Great! Amazing! Good work with the plot using ggplot. Well done!

Again, if I were to be strict with this task, my specific requirement was to create a function. Your script here is a good example of how to script the creation of a ggplot. I was wondering if you would like a challenge in converting this to a function?